ccm_web/client/src/components/CreateSectionWidget.tsx

```diff
@@ -54,18 +56,26 @@ function CreateSectionWidget (props: CreateSectionWidgetProps): JSX.Element {
             props.onSectionCreated(newSections[0])
             setNewSectionName('')
             enqueueSnackbar('Section was created!', { variant: 'success' })
-          }).catch(() => {
-            enqueueSnackbar('Error adding section', {
-              variant: 'error'
-            })
+          }).catch((error: unknown) => {
+            enqueueSnackbar(
+              <APIErrorMessage
+                context='creating the section'
+                error={error instanceof Error ? error : undefined}
```

Comment:
This will almost always be an `Error`. In the case where it isn't, the user will see a generic message, and we can ask them to look in the Web console for more information if we need to investigate.